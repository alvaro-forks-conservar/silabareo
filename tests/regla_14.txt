balaán
tonteáis
loáis
aéreo
toreé
poético
faraónico
torpedeó
loó
ahíto
oleína
mohín
baúl
rehúso
mohúr