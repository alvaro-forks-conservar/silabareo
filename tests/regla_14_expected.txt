ba-la-án
ton-te-áis
lo-áis
a-é-re-o
to-re-é
po-é-ti-co
fa-ra-ó-ni-co
tor-pe-de-ó
lo-ó
a-hí-to
o-le-í-na
mo-hín
ba-úl
re-hú-so
mo-húr